src/main/java/org/jenkinsci/plugins/workflow/multibranch/SCMVar.java

```diff
@@ -97,7 +97,7 @@
         } else {
             SCMHead head = branch.getHead();
             FlowExecutionOwner owner = ((WorkflowRun) build).asFlowExecutionOwner();
-            TaskListener listener = owner != null ? owner.getListener() : TaskListener.NULL;
+            TaskListener listener = owner.getListener();
```

Comment:
`asFlowExecutionOwner` is `@NonNull` in recent versions of `workflow-job`, so SpotBugs didn't like the redundant null check.